kpops/component_handlers/helm_wrapper/helm.py

```diff
@@ -31,11 +31,14 @@
 
 @final
 class Helm:
+    _version: Version = None
```

Comment:
can we do this directly and remove it from init?
```suggestion
    _version: Version = Helm.get_version()
```

Reply:
I tried and failed, it seems like we can't init static variables from static methods from the same class, I get the following error:

```
NameError: name 'Helm' is not defined
```